src/components/marker.js

```diff
@@ -54,7 +54,7 @@ export default class Marker extends Component {
     };
 
     return createElement('div', {
-      className: 'map-control-marker',
+      className: 'mapboxgl-marker',
```

Comment:
This one is not dependent on the mapbox stylesheet because it's already in-lined (position: absolute). The change of class name is just to align with what's used in the other controls.
The Popup control on the other hand has ~150 lines of css. I'm not convinced that it's worth the effort to recreate it. Alternatively we could:
- Offer our own stylesheet built on top of the mapbox-gl one (users still need to import it)
- Embed the stylesheet in JavaScript (will be harder for users to override)